homeassistant/components/velbus/climate.py

```diff
@@ -79,3 +85,11 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         """Set the new preset mode."""
         await self._channel.set_preset(PRESET_MODES[preset_mode])
         self.async_write_ha_state()
+
+    @api_call
+    async def async_set_hvac_mode(self, **kwargs: str) -> None:
+        """Set the new hvac mode."""
+        if (mode := kwargs.get("hvac_mode")) is None:
+            return
```

Comment:
This is impossible, the service schema requires `hvac_mode` to be a valid `HVACMode`.
OTOH, `HVACMode` allows other modes than `cool` and `heat` and I think we need to guard against *that*.

Reply:
removed the if none ..